src/test/java/com/scottlogic/competitionEngine/application/user/UserServiceImplTests.java

```diff
@@ -0,0 +1,32 @@
+package com.scottlogic.competitionEngine.application.user;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
```

Comment:
Maybe more a @FrancisD-SL question, but is there a reason we're using junit4 over 5?